app/helpers/ems_storage_helper/textual_summary.rb

```diff
@@ -35,7 +35,7 @@ def textual_group_topology
   #
   def textual_provider_region
     return nil if @record.provider_region.nil?
-    {:label => _("Region"), :value => @record.description}
+    {:label => _("Region"), :value => @record.try(:description)}
```

Comment:
I wonder why this was checking `provider_region` but presenting `description`?

I'm assuming the `.try()` here is because you hit a `@record` that didn't have an instance method `:description`?  I see the Openstack Cinder/Swift storage managers duplicate the typical class-method `.description` so maybe that's what this was expecting to work with?

Either way I'm not sure that showing e.g. `"Cinder"` or `"Swfit"` for the `Region` makes sense, wonder if we should just be showing `@record.provider_region` here

Reply:
ok i can change it to @record.provider_region, it is checking for that but showing description which is weird.

Reply:
Yeah my guess is this was an implicit "only openstack storage managers have a provider_region", still not sure why we'd show description not region for the Region :shrug: